modules/choosing-container-build-tools.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * architecture/understanding-openshift-development.adoc
+
+[id="choosing-container-build-tools_{context}"]
```

Comment:
Is this something that users are going to do for themselves or something that their teams will have already decided for them? Also, can we anchor this in the OCP context?